app/api/event_image_sizes.py

```diff
@@ -4,15 +4,32 @@
 from app.api.schema.image_sizes import EventImageSizeSchema
 from app.models import db
 from app.models.image_size import ImageSizes
-
+from app.api.helpers import db as db_helper
 
 class EventImageSizeDetail(ResourceDetail):
     """
     Event Image_size detail by id
     """
     @classmethod
     def before_get(self, args, kwargs):
-        kwargs['id'] = 1
+        event_image_size, is_created = db_helper.get_or_create(ImageSizes, type='event-image')
+        if is_created:
+            event_image_size.full_width = 1300,
+            event_image_size.full_height = 500
+            event_image_size.full_aspect = True
+            event_image_size.full_quality = 80,
+            event_image_size.icon_width = 75
+            event_image_size.icon_height = 30
+            event_image_size.icon_aspect = True,
+            event_image_size.icon_quality = 80
+            event_image_size.thumbnail_width = 500
+            event_image_size.thumbnail_height = 200,
+            event_image_size.thumbnail_aspect = True
+            event_image_size.thumbnail_quality = 80
+            event_image_size.logo_width = 500
+            event_image_size.logo_height = 200
```

Comment:
Umm, pass this in get_or_create itself

Reply:
@iamareebjamal - When you change one field value, it will again create because default values aren't matched.